packages/amplify-cli/src/lib/input-params-manager.js

```diff
@@ -26,10 +26,12 @@ function normalizeKey(key) {
 }
 
 function normalizeValue(key, value) {
-  if (key === 'app') {
-    return value;
+  let normalizedValue = value;
```

Comment:
Instead of mutating a variable I suggest we could do this

`try {
   return JSON.parse(value);
} catch (e) {
  // do nothing
}
return value;`

Reply:
We can't do it. 
Basically what this function does is that, if the import string can be parsed as JSON, it will parse it, else it will return as is.